cpu/atmega_common/periph/i2c.c

```diff
@@ -32,9 +32,6 @@
 #include "debug.h"
 #define ENABLE_DEBUG      (0)
 
-/* guard file in case no I2C device is defined */
```

Comment:
This looks unrelated, weird.

Reply:
yes, no idea how that commit sneaked in. I squashed it out.